docs/root/intro/arch_overview/other_protocols/redis.rst

```diff
@@ -27,6 +27,7 @@ The Redis project offers a thorough reference on partitioning as it relates to R
 * Prefix routing.
 * Separate downstream client and upstream server authentication.
 * Request mirroring for all requests or write requests only.
+* Control :ref:`read requests routing<envoy_api_field_config.filter.network.redis_proxy.v2.RedisProxy.ConnPoolSettings.read_policy>`.
```

Comment:
Per other comment please clarify that this only works with Redis Cluster. Same in the release notes.